classes/local/report/objectfs_report.php

```diff
@@ -166,6 +166,7 @@ public static function get_report_types() {
             'location',
             'log_size',
             'mime_type',
+            'tag_count',
```

Comment:
This is kinda baked into the flexible table itself and not something you are meant to be able to change - see https://github.com/moodle/moodle/blob/6751ba650055ec1146df5e05b12972fa234b37f8/lib/tablelib.php#L1025

To make it a tiny bit better I've added the header for each table above it - it's a bit redundant for the others (since they have the name of the table in the table anyway) but it's the 'least bad' solution i'm aware of:
![image](https://github.com/user-attachments/assets/d3884c5f-1c0d-47b4-9650-1cdb7e23647e)

Reply:
Thanks Matt, the header resolves the main issue of the confusing message. I was thinking it may be possible to add some logic in get_report_types() to decide whether you add 'tag_count', but the header change should be enough.